### PR TITLE
Pap 138 install prettier linter and jest on intracom webapp

### PR DESCRIPTION
**Pull request by Amenu-sh**
## Problem
Code Quality and standardize

## Solution
Test the configuration and resolve any issues.

## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## How Has This Been Tested?
Implemented Prettier, linter, jest, Commitlint, and husky.

## Checklist:
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .husky/commit-msg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```
Comment (oestrada1001) [inline .husky/commit-msg:3]: **Actionable**
Line 2 and 3 are not required with the latest version of husky

File: public/zoom 2.png
Comment (oestrada1001) [inline public/zoom 2.png:1]: **Actionable**
Please remove these images if they aren't being used.

File: public/service 2.png
Comment (oestrada1001) [inline public/service 2.png:1]: **Actionable**
Please remove this image if it isn't being used.

File: public/preporganization 2.png
Comment (oestrada1001) [inline public/preporganization 2.png:1]: **Actionable**
Please remove this image if it isn't being used.

File: public/jira 2.png
Comment (oestrada1001) [inline public/jira 2.png:1]: **Actionable**
Please remove this image if it isn't being used.

File: public/gitlab 2.png
Comment (oestrada1001) [inline public/gitlab 2.png:1]: **Actionable**
Please remove this image if it isn't being used.

File: public/github 2.png
Comment (oestrada1001) [inline public/github 2.png:1]: **Actionable**
Please remove this image if it isn't being used.

File: src/app/Successful/page.tsx
Comment (oestrada1001) [inline src/app/Successful/page.tsx:1]: **Actionable**
Let's make sure the directory name is 'successful' with a lowercase 's' when we merge to main.

File: src/app/layout.tsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+import { Inter } from 'next/font/google';
+import './globals.css';
 
-const inter = Inter({ subsets: ["latin"] });
+const inter = Inter({ subsets: ['latin'] });
 
 export const metadata: Metadata = {
-  title: "Intracom",
-  description: "",
+  title: 'Create Next App',
```
Comment (oestrada1001) [inline src/app/layout.tsx:8]: **Actionable**
Please revert title and and description.
```
Title: 'Intracom'
Description: ''
```

**Review by oestrada1001 — CHANGES_REQUESTED** [review]

File: public/zoom 2.png
Comment (Amenu-sh) [reply public/zoom 2.png:1]: images are used inside the Preparing-workspace component.

File: public/preporganization 2.png
Comment (Amenu-sh) [reply public/preporganization 2.png:1]: doubled images are being removed.

**Review by oestrada1001 — APPROVED** [review]
Looks great! Great job @Amenu-sh!

**Pull request merged**